### PR TITLE
feat: adds the basic routing with screen

### PR DESCRIPTION
**Pull request by FarhanMobashir**
- adds the basuc routing using react router
- adds the styling of all the screen by adding some css and mostly using
  my own component library
- made some react component using my cl

### CODE REVIEW

**Review by GuptaEesh — COMMENTED** [review]
Everything just looks picture perfect.
Great use of component hierarchy
Code just flows into perfection.
Great work.

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,28 @@
+import { Route, Routes } from "react-router-dom";
+import { AppLayout } from "./components/AppLayout";
+import { AuthScreen } from "./screens/AuthScreen";
+import { CartScreen } from "./screens/CartScreen";
+import { HomeScreen } from "./screens/HomeScreen";
+import { ProductsScreen } from "./screens/ProductsScreen";
+import { WishlistScreen } from "./screens/WishListScreen";
+import Mockman from "mockman-js";
```
Comment (Hamdrive) [inline src/App.js:8]: Within your screen directory, you can keep a index file, from where you can export all these screens, thus you would only have to do one import here, keeping your code even neat!

**Review by Hamdrive — COMMENTED** [review]
Really amazing and clean code written, liked the code structure and very little to no clutter
Good use of breaking down logic into individual functions/components
Enjoyed reviewing this PR!

File: src/App.js (hunk 1 above)
Comment (FarhanMobashir) [reply src/App.js:8]: Yes hamza. Thanks for pointing out. I'll refactor it in the next pr

**Pull request merged**